server/middleware/setUpCsrf.ts

```diff
@@ -14,7 +14,7 @@ export default function setUpCsrf(): Router {
       // By default, csrf-sync uses x-csrf-token header, but we use the token in forms and send it in the request body, so change getTokenFromRequest so it grabs from there
       getTokenFromRequest: req => {
         // eslint-disable-next-line no-underscore-dangle
-        return req.body._csrf
+        return req.body?._csrf
```

Comment:
The req.body property returns undefined when the body has not been parsed. In Express 4, it returns {} by default.

https://expressjs.com/en/guide/migrating-5.html#req.body